Readme.md

```diff
@@ -0,0 +1,494 @@
+
+```groovy
+dependencies {
+    implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.6.4'
+    implementation 'org.mongodb:mongodb-driver-kotlin-coroutine:4.10.0-SNAPSHOT'
+}
+```
+
+And now we can connect with [MongoDB Atlas][1] using Kotlin driver.
+
+---------------------
+
+## Connecting to database
+
+To connect with the database, we first need the `Connection URI` that can be found by pressing connect to cluster at
+[Atlas account][3], as shown below.
+
+![image](https://images.contentstack.io/v3/assets/blt39790b633ee0d5a7/blt1d92c6f1c6654b04/648793839625e14516b3657c/68747470733a2f2f6d6f6e676f64622d6465766875622d636d732e73332e75732d776573742d312e616d617a6f6e6177732e636f6d2f436f6e6e656374696f6e5f5552495f666439393037653262642e706e67.png)
+
+For more details, you can also refer to this [documentation](https://www.mongodb.com/docs/guides/atlas/connection-string/).
+
+With connection URI available, the next step is to create a Kotlin file, `Setup.kt` where we would be writing the code for connecting to [MongoDB
+Atlas][1].
+
+![Setup.kt file](https://images.contentstack.io/v3/assets/blt39790b633ee0d5a7/bltdc533d1983ce2f87/6488261e8b23a52669052cee/Screenshot_2023-06-13_at_09.17.29.png)
+
+Connection with our database can be split into two steps first we create MongoClient instance using `Connection URI`.
+
+```kotlin
+val connectionString = "mongodb+srv://mohitsharma:<enter your password>@cluster0.sq3aiau.mongodb.net/?retryWrites=true&w=majority"
+val client = MongoClient.create(connectionString = connectString)
+```
+
+And then with that, we can connect to the database, `sample_restaurants` which is a restaurant [sample dataset][2] in Atlas ,that help developer
+learn and explore MongoDB in a realistic environment and would also help us build more realistic application.
+
+```kotlin
+val databaseName = "sample_restaurants"
+val db: MongoDatabase = client.getDatabase(databaseName = databaseName)
+```
+
+And since it's not good practice to hardcode `connectionString` in code, therefore let's move it outside. We would be using environment variables
+for simplicity, but you can use Vault, build configuration variable or CI/CD environment variable whatever suites you well.
+
+To add environment variables use `Modify run configuration` that can be found by right-clicking on the file.
+
+![add environment variable](https://images.contentstack.io/v3/assets/blt39790b633ee0d5a7/blt8438bbb2ace0979e/64882bac83c7fb1a685f3d1b/Screenshot_2023-06-13_at_09.38.49.png)
+
+And after everything done our final code would look like this
+
+```kotlin
+suspend fun setupConnection(
+    databaseName: String = "sample_restaurants",
+    connectionEnvVariable: String = "MONGODB_URI"
+): MongoDatabase? {
+    val connectString = if (System.getenv(connectionEnvVariable) != null) {
+        System.getenv(connectionEnvVariable)
+    } else {
+        "mongodb+srv://mohitsharma:gq0Sj8aUXucHQtc2@cluster0.sq3aiau.mongodb.net/?retryWrites=true&w=majority"
+    }
```

Comment:
To avoid confusion and promote bad practice, I would remove this even knowing that this is an example.

Reply:
Firstly thank you for flagging this, this got leaked accidentally due to the last min of code refactoring and copy-pasting. 

I have mentioned just below it and highlighted that it's not a good practice to do this and at the same time don't want to add a barrier for first-time users to run the code. Will also add a code comment to twice highlight not a good practice.